docs/Installing_and_RunningTheHPCCPlatform/Inst-Mods/hpcc_ldap.xml

```diff
@@ -225,7 +225,7 @@
         </listitem>
 
         <listitem>
-          <para>Select the security Plugin for each service that requires a
+          <para>Select the security Plug-in for each service that requires a
```

Comment:
But you have introduced the incorrect version with this pull request - that makes it inside the scope!